packages/xrpl/test/client/submitAndWait.test.ts

```diff
@@ -0,0 +1,44 @@
+import { XrplError } from '../../src'
```

Comment:
This file (`packages/xrpl/test/integration/submitAndWait.test.ts`) is also relevant for adding this test case. Why did you create a new file?

Reply:
It's easier to test this sort of thing with a unit test where I can control the fixture as opposed to with an integration test.